tests/containers/targetcli/Containerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/centos/centos:stream9
+FROM registry.fedoraproject.org/fedora-minimal:42
```

Comment:
They're equivalent but I consider `quay.io/fedora/fedora-minimal:42` more canonical at this point.